stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -297,11 +305,11 @@ function getAuthTokenForOAuth2PasswordGrant(PasswordGrantConfig grantTypeConfig,
     }
 }
 
-# Process the auth token for OAuth2 client credentials grant.
+# Process the OAuth2 token for client credentials grant type.
 #
-# + grantTypeConfig - Client credentials grant configurations
+# + grantTypeConfig - Client credentials grant type configurations
 # + oauth2CacheEntry - OAuth2 cache entry
-# + return - Auth token or `Error` if an error occurred during the HTTP client invocation or validation
+# + return - OAuth2 token or `oauth2:Error` if an error occurred during the HTTP client invocation or validation
```

Comment:
```suggestion
# + return - OAuth2 token or `oauth2:Error` if an error occurred during the HTTP client invocation or validation
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - OAuth2 token, or else `oauth2:Error` if an error occurred while HTTP client invocation or validation
```